package.json

```diff
@@ -17,7 +17,7 @@
     "cypress": "7.0.0",
     "dayjs": "^1.10.4",
     "ory-prettier-styles": "1.1.1",
-    "prettier": "2.2.1",
+    "prettier": "^2.2.1",
```

Comment:
The problem is actually this change, in the CI it actually installed `2.3.1` instead of `2.2.1`. That's why the styles did not match up.